cmd/podman/networks/create.go

```diff
@@ -78,6 +78,10 @@ func networkCreateFlags(cmd *cobra.Command) {
 	_ = cmd.RegisterFlagCompletionFunc(subnetFlagName, completion.AutocompleteNone)
 
 	flags.BoolVar(&networkCreateOptions.DisableDNS, "disable-dns", false, "disable dns plugin")
+
+	dnsserverFlagName := "network-dns-servers"
```

Comment:
I would call this just `dns` like it is for `podman create/run`, I bet @rhatdan will agree since he does not like options with extra dashes in the name.

Reply:
changed